client/js/keybinds.js

```diff
@@ -103,6 +104,54 @@ Mousetrap.bind([
 });
 
 const inputTrap = Mousetrap(input.get(0));
+
```

Comment:
At the risk of keybinds getting too big, can we stick this in a different file?

I actually think that the keybindings should probably go in next to code similar to it rather than having 1 massive folder defining all the keybindings, but at the very least, can we just have the history code in a different file, and just have the keybinding here?

Reply:
The only real reason I put it here is because I need access to `inputTrap`.

Reply:
We can grab that from elsewhere, can't we? Is there any disadvantage to just instantiating it in another module again? I assume it caches.

Reply:
Mousetrap doesn't cache and it would bind a separate event again. What about moving code that uses `inputTrap` to separate file? Call it `inputHandler` or something.

Reply:
Euch. That feels like just kicking the can down the road. Alright, keep it here for now and we can rethink how we structure these later on.